internal/main.go

```diff
@@ -33,8 +35,26 @@ func RunTests(opts RunTestsOpts) {
 	// Exclude by package prefix
 	testPkgs := allPkgs
 	if len(opts.Excludes) > 0 {
-		testPkgsStr := shCmd("grep", shArgs{"-Ev", getExcludePattern(opts.Excludes)}, allPkgsStr)
-		testPkgs = splitLines(testPkgsStr)
+		testPkgs = []string{}
```

Comment:
Also, if we move this to a function it's easy to put it under a test.

Reply:
Done